.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
What does `c_cpp` mean in the filename?

Reply:
this is vscode configuration to enable intellisence. That's just the name vscode is expecting to find

Reply:
We should be excluding .vscode folder

.gitignore should have:
# VS Code
.vscode/
.factorypath

Reply:
fine